### ISSUE TITLE
Named scenario processing errors should be visible

### ISSUE THREAD

**Issue by jshook** [nosqlbench/nosqlbench#215]
During NB bootstrapping, named scenarios are mapped out for the valid named scenario command line forms. When there is an error in this processing, specifically during yaml parsing, the errors are silent.

The logging system needs to be configured and activated *before* named scenario mapping.

**Comment by dave2wave** [comment]
I'm addi g this to an Initiative. Should it validated as current it is certainly part of logging and error handling

**Comment by dave2wave** [comment]
I moved this to **NBAdvisor**

**Comment by dave2wave** [comment]
Logging is started here: https://github.com/nosqlbench/nosqlbench/blob/main/nb-engine/nb-engine-cli/src/main/java/io/nosqlbench/engine/cli/NBCLI.java#L81
And configured here: https://github.com/nosqlbench/nosqlbench/blob/main/nb-engine/nb-engine-cli/src/main/java/io/nosqlbench/engine/cli/NBCLI.java#L159-L195
Here is where the scenario yaml parsing is triggered: https://github.com/nosqlbench/nosqlbench/blob/main/nb-engine/nb-engine-cli/src/main/java/io/nosqlbench/engine/cli/NBCLI.java#L225

In conclusion this was fixed at some point in the past.
